console/console-init/ui/src/modules/iot-device-detail/components/DeviceDetailHeader/DeviceDetailHeader.tsx

```diff
@@ -229,7 +230,7 @@ const DeviceDetailHeader: React.FunctionComponent<IDeviceDetailHeaderProps> = ({
       <DeviceDetailLayout />
       <SplitItem isFilled />
       <SplitItem className={css(styles.kebab_toggle_margin)}>
-        <SwitchWithToggle
+        <Switch
```

Comment:
why did you change SwitchWithToggle to Switch ?

Reply:
SwitchWithToggle does not wait till action from wizard is taken, it changes state on its own.